pkg/configurator/client.go

```diff
@@ -55,6 +55,9 @@ const (
 
 	// enablePrivilegedInitContainer is the key name used to specify whether init containers should be privileged in the ConfigMap
 	enablePrivilegedInitContainer = "enable_privileged_init_container"
+
+	// enableTicker is a flag to configure ticker for regular proxy broadcast updates
+	enableTicker = "enable_ticker"
```

Comment:
`enable_ticker` makes it sound like it is a bool value. The word `ticker` also does not have a clear meaning.

I would call this `envoy_resync_period` or `sidecar_resync_interval`...
Also make it clear in the comment what value we are expecting to be in it - might help the reader at some point in the future understand.

Reply:
done, reanamed to `config_resync_interval`